crates/threads-xform/tests/basic.wat

```diff
@@ -76,41 +76,56 @@
     global.set 1
     global.get 1
     call $__wasm_init_tls)
-  (func $__wbindgen_thread_destroy (type 0)
-    global.get 1
-    i32.const 128
-    call $__wbindgen_free
-    i32.const -2147483648
-    global.set 1
-    i32.const 393216
-    global.set 2
-    loop  ;; label = @1
-      i32.const 327684
-      i32.const 0
-      i32.const 1
-      i32.atomic.rmw.cmpxchg
-      if  ;; label = @2
+  (func $__wbindgen_thread_destroy (type 3) (param i32 i32)
+    (local i32 i32)
```

Comment:
:warning: These are a bit annoying. I couldn't find a way in walrus to generate references to function parameters without creating spurious locals.